e2e/src/ui/selectors/sideBarSelectors.ts

```diff
@@ -1,8 +1,8 @@
 import { Attributes, Tags } from '@/e2e/src/ui/domData';
 
 export const SideBarSelectors = {
-  chatBar: '[data-qa="sidebar"].fixed.left-0',
-  promptBar: '[data-qa="sidebar"].fixed.right-0',
+  chatBar: '[data-qa="chatbar"]',
+  promptBar: '[data-qa="promptbar"]',
   folder: '[data-qa="folder"]',
```

Comment:
It would be better to add `group/chatbar` and `group/promptbar`  [on this line](https://github.com/epam/ai-dial-chat/blob/development/src/components/Sidebar/Sidebar.tsx#L78) according to [this issue](https://github.com/epam/ai-dial-chat/issues/255) to use more semantic css classes. And then use it here